pkg/userdata/ubuntu/userdata.go

```diff
@@ -197,7 +197,7 @@ write_files:
 
 - path: "/etc/apt/sources.list.d/docker.list"
   permissions: "0644"
-  content: deb [arch=amd64] https://download.docker.com/linux/ubuntu xenial stable
+  content: deb [arch=amd64] https://download.docker.com/linux/ubuntu bionic stable
```

Comment:
Not really happy with that because we do not always use bionic

Reply:
We switched to bionic, didn't we?

Reply:
Yes, we do use bionic now.